ceph.spec.in

```diff
@@ -339,6 +339,8 @@ BuildRequires:  libpciaccess-devel
 BuildRequires:  lksctp-tools-devel
 BuildRequires:  ragel
 BuildRequires:  systemtap-sdt-devel
+BuildRequires:  libasan
+BuildRequires:  libubsan
```

Comment:
might want to remove the `BuildRequires` in the following two lines? they duplicate with what you are adding.

Reply:
Thank you for the suggestion Kefu, updated.